live-examples/css-examples/basic-user-interface/outline.html

```diff
@@ -0,0 +1,45 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="outline">
+
+    <div class="example-choice">
+        <pre><code id="example_one" class="language-css">outline: solid;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_two" class="language-css">outline: dashed red;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_three" class="language-css">outline: 1rem solid;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_four" class="language-css">outline: thick double #32a1ce;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code id="example_five" class="language-css">outline: 3mm ridge rgb(170, 50, 220, .6);
```

Comment:
mm is not a common unit on the web, unless you are specifying a print stylesheet so, I would recommend using px here instead.